core/src/main/scala/dev/profunktor/auth/middleware.scala

```diff
@@ -18,7 +18,9 @@ object JwtAuthMiddleware {
     val dsl = new Http4sDsl[F] {}; import dsl._
 
     val onFailure: AuthedRoutes[String, F] =
-      Kleisli(req => OptionT.liftF(Forbidden(req.authInfo)))
+      Kleisli {
+        case _ as authInfo => OptionT.liftF(Forbidden(authInfo))
```

Comment:
You can leave it as it was and just use `context` instead:

```scala
Kleisli(req => OptionT.liftF(Forbidden(req.context)))
```

Reply:
I agree